workspace/api/openfdd_bridge/pooge_service.py

```
@@ -0,0 +1,247 @@
+
+    for item in targets:
+        action = item.get("action")
+        label = str(item.get("label") or action)
+        path_raw = str(item.get("path") or "")
+
+        if action == "linux_update":
+            audit.append(f"linux_update: {'skipped (dry-run)' if req.dry_run else 'requested'}")
+            if not req.dry_run:
+                try:
+                    proc = subprocess.run(
+                        ["apt-get", "update"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)